### PR TITLE
Release v1.6.0

### PR DESCRIPTION
**Pull request by smurf-U**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -7,6 +7,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+## [1.6.0] - 2022-07-07
+
+### Added
+
+- Add sample example for AIOHTTP Framework
+
+### Changed
+
+- Improve AIOHTTP middleware with APM support of Jinja Template, SqlAlchemy
+- Add and Improve code docstring and code refactoring
```
Comment (kyrylo) [inline changelog.md:19]: Let's omit code refactoring entries. They bring no value to our users.

File: changelog.md [hunk 2]
```diff
@@ -7,6 +7,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Improve README file
+- Bump pycares from 4.1.2 to 4.2.0 in /examples/aiohttp
```
Comment (kyrylo) [inline changelog.md:21]: Nobody cares about pycares being updated here 😁

File: changelog.md [hunk 3]
```diff
@@ -7,6 +7,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Improve AIOHTTP middleware with APM support of Jinja Template, SqlAlchemy
+- Add and Improve code docstring and code refactoring
+- Improve README file
+- Bump pycares from 4.1.2 to 4.2.0 in /examples/aiohttp 
+- Bump ujson from 5.3.0 to 5.4.0 in /examples/sanic
+- Bump django from 3.2.13 to 3.2.14 in /examples/django
+
+### Fixed
+
+- Fixed AIOHTTP middleware for Error Monitoring
```
Comment (kyrylo) [inline changelog.md:27]: Describe what was fixed.

File: changelog.md [hunk 4]
```diff
@@ -7,6 +7,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Added
+
+- Add sample example for AIOHTTP Framework
+
+### Changed
+
+- Improve AIOHTTP middleware with APM support of Jinja Template, SqlAlchemy
+- Add and Improve code docstring and code refactoring
+- Improve README file
```
Comment (kyrylo) [inline changelog.md:20]: Why would anyone care?

**Review by kyrylo — CHANGES_REQUESTED** [review]
Let's not mention internal changes in our CHANGELOG. The CHANGELOG is not meant to be a duplicate of the git logs. We must talk to the user and explain what value we added or what public APIs we've changed.

**Comment by smurf-U** [comment]
PTAL

**Review by kyrylo — APPROVED** [review]

File: changelog.md [hunk 5]
```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.6.0] - 2022-07-07
```
Comment (kyrylo) [inline changelog.md:10]:
```suggestion
## [1.6.0] - 2022-07-08
```

**Pull request merged**